src/test/server/database/test_database.py

```diff
@@ -49,38 +52,17 @@ def setUp(self) -> None:
                                         WorkMachineResources(ResourceAllocation(12, 32, 12)))
         self.work_machine2 = WorkMachine("machi2", WorkMachineState.ONLINE,
                                          WorkMachineResources(ResourceAllocation(2, 2, 2)))
+        self.work_machine3 = WorkMachine("machi1", WorkMachineState.ONLINE,
```

Comment:
Is the UID intentionally the same as machine1 ?

Reply:
yes, I just wanted to test when I update wm1 if it is going to break